Build/phpmd.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<ruleset name="phpList">
+    <description>
+        PHPMD rules for Tea
+    </description>
+
+    <!-- The commented-out rules will be enabled once the code does not generate any warnings anymore. -->
+
+    <rule ref="rulesets/cleancode.xml/BooleanArgumentFlag"/>
+    <rule ref="rulesets/cleancode.xml/StaticAccess"/>
+
+    <rule ref="rulesets/codesize.xml/CyclomaticComplexity"/>
+    <rule ref="rulesets/codesize.xml/NPathComplexity"/>
+    <rule ref="rulesets/codesize.xml/ExcessiveMethodLength"/>
+    <!--<rule ref="rulesets/codesize.xml/ExcessiveClassLength"/>-->
+    <rule ref="rulesets/codesize.xml/ExcessiveParameterList"/>
+    <rule ref="rulesets/codesize.xml/ExcessivePublicCount"/>
+    <rule ref="rulesets/codesize.xml/TooManyFields"/>
+    <!--<rule ref="rulesets/codesize.xml/TooManyMethods"/>-->
+    <rule ref="rulesets/codesize.xml/TooManyPublicMethods"/>
+    <!--<rule ref="rulesets/codesize.xml/ExcessiveClassComplexity"/>-->
```

Comment:
Please let's check which of these commented-out rules actually create a warning. For those that currently create a warning, we should create a ticket to fix the code, and then either raise the limit or disable the corresponding rule.